spec/models/application_choice_spec.rb

```diff
@@ -12,4 +12,12 @@
       expect(application_form.updated_at).not_to eql(original_time)
     end
   end
+
+  describe '#create' do
+    it 'starts in the "unsubmitted" state' do
+      application_choice = ApplicationChoice.create!(application_form: create(:application_form))
+
+      expect(application_choice).to be_unsubmitted
+    end
+  end
```

Comment:
Might also want to test that it's possible to override the default, e.g. create a completed application (because you are using `||=` in the implementation).